packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -10,6 +10,7 @@ import { DEFAULT_LOW_KARMA_THRESHOLD } from '../../lib/collections/posts/views'
 import StarIcon from '@material-ui/icons/Star'
 import DescriptionIcon from '@material-ui/icons/Description'
 import MessageIcon from '@material-ui/icons/Message'
+import ChangeHistoryIcon from '@material-ui/icons/ChangeHistory'
```

Comment:
I wasn't sure what the best icon to use was - I think the pencil might make more sense, but also the pencil looks like it should do something when you click on it so that might be confusing.